homeassistant/components/twitch/sensor.py

```diff
@@ -96,10 +96,7 @@ def entity_picture(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        attr = {
-            ATTR_FRIENDLY_NAME: self._channel.display_name,
-        }
-        attr.update(self._statistics)
+        attr = self._statistics
```

Comment:
Careful here, I haven't seen the rest of the code but you're now mutating `self._statistics` instead of a copy. So now if the state is no longer streaming, the game/title attributes might still be present.

Reply:
You are correct, read up on it and now use copy.deepcopy().